src/scenes/registrations/containers/registrations_container.js

```diff
@@ -114,7 +114,7 @@ class RegistrationsContainer extends Component {
         },
         {
           headers: {
-            'X-Timezone-Offset': new Date().getTimezoneOffset(),
+            'X-Timezone': Intl.DateTimeFormat().resolvedOptions().timeZone,
```

Comment:
@fcmatteo we should probably check if this API is available in the browser, else it might crash.